### ISSUE TITLE
Local storage backups trigger checksum mismatch on verify 

### ISSUE THREAD

**Issue by TryToWork** [thelastpickle/cassandra-medusa#105]
Hello,

I am new user of Medusa, i need to use it for sav my cassandra cluster (three nodes for this test).
I decide to use NFS for begin.

I execute on each node (not parrallel , sequecially) : 
medusa backup --backup-name="first_try" --mode=full

No errors !

I check the status every thing is good 👍 

- Started: 2020-04-07 10:51:37, Finished: 2020-04-07 10:54:31
- 3 nodes completed, 0 nodes incomplete, 0 nodes missing
- 2512 files, 182.38 MB

But when i want verify the backup : 
I have checksum problem and an error of manifest ...

.........
  - [node1/first_try/data/system/sstable_activity-5a1ff267ace03f128563cfae6103c65e/md-11391-big-Summary.db] Wrong checksum
................
RuntimeError: Manifest validation failed

Do you an idea for resolve this problem please?

**Comment by adejanovski** [comment]
Hi @TryToWork,

could you provide us with the hashes that are reported by `medusa verify` for this file? (the computed one and the one from the manifest).

**Comment by TryToWork** [comment]
First thank you for your answer.

But in the meantime I deleted three backups folders + index folder and this time it's work i can do a backup full.

The command verify is good and the manifest are good ....
- Completion: OK!
- Manifest validated: OK!!

But if i launch an other backup full with an other name, the both manifest become invalid when i do a verify...

So first i think we can't have two or more backup full but i tried one backup full + one differential 
 and it's the same result ...

And only one backup differrential not work too ....

I have to leave, tomorrow i will upload the files...

**Comment by TryToWork** [comment]
Hello,

The logs you ask and the command i pasted .
[liste commandes.txt](https://github.com/thelastpickle/cassandra-medusa/files/4450401/liste.commandes.txt)

It's like I described above the first backup is ok but when I launch the second the two manifest become invalid.

**Comment by oliviercwa** [comment]
Hello,
I am having exactly the same issue using Medusa 0.5.1. I have a 3 node cluster in Azure with 100% data replication and a shared backup drive accessible via SMB 3.0.

This is the Medusa configuration used:
```
[monitoring]
monitoring_provider = local
[storage]
bucket_name = backups
multi_part_upload_threshold = 104857600
concurrent_transfers = 1
max_backup_age = 365
transfer_max_bandwidth = 50MB/s
max_backup_count = 1000
storage_provider = local
base_path = /backupdrive/cassandra
[cassandra]
start_cmd = systemctl start cassandra-cluster.service
stop_cmd = systemctl stop cassandra-cluster.service
config_file = /etc/cassandra/default.conf/cassandra.yaml
```

On each of the node, I ran:
```
sudo /usr/local/bin/medusa backup
```

When I check the status of the backup, I get:
```2020042315
- Started: 2020-04-23 15:29:18, Finished: 2020-04-23 15:35:30
- 3 nodes completed, 0 nodes incomplete, 0 nodes missing
- 2523 files, 23.93 MB
```

When I use the verify function, I get Completion OK followed by lots of CHECKSUM errors:
```
Validating 2020042315 ...
- Completion: OK!
```
Followed by a bunch of ERRORS like this one:
```
[2020-04-23 15:51:34,490] ERROR: Expected d9fe9eebbd5d6bdedcf79d387dbdf9e36f3673471ed35e3b but got 3cf0b7ebc4695c0944e738c6081b1f87 for dev-cas-i1.internal.cloudapp.net/data/document/credential-6e5f0ce078b811eab176094b5cadb1c0/md-1-big-Data.db
```
Followed by many lines like this one (essentially all keyspaces and all tables...):
```
- [dev-cas-i1.internal.cloudapp.net/data/document/rule-4df1f6c078b811eab176094b5cadb1c0/md-15-big-Filter.db] Wrong checksum
- [dev-cas-i1.internal.cloudapp.net/data/document/rule-4df1f6c078b811eab176094b5cadb1c0/md-15-big-Statistics.db] Wrong checksum
- [dev-cas-i1.internal.cloudapp.net/data/document/rule-4df1f6c078b811eab176094b5cadb1c0/md-15-big-Index.db] Wrong checksum
```
And at the end:
```  
File "/usr/local/bin/medusa", line 11, in <module>
    sys.exit(cli())
  File "/usr/local/lib64/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib64/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib64/python3.6/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib64/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib64/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib64/python3.6/site-packages/click/decorators.py", line 73, in new_func
    return ctx.invoke(f, obj, *args, **kwargs)
  File "/usr/local/lib64/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/medusa/medusacli.py", line 232, in verify
    medusa.verify.verify(medusaconfig, backup_name)
  File "/usr/local/lib/python3.6/site-packages/medusa/verify.py", line 53, in verify
    raise RuntimeError("Manifest validation failed")
RuntimeError: Manifest validation failed
```

Is this an known issue? Am I doing something incorrectly?

**Comment by adejanovski** [comment]
Libcloud doesn't compute a hash based on file content but on modification time for the local storage provider : https://github.com/apache/libcloud/blob/v2.8.0/libcloud/storage/drivers/local.py#L178

This can lead to the behavior above if the SAN or the FS changes modification times upon copy, not retaining the original attributes.

We'll disable hash comparisons for now and work on a proper fix at a later time.

**Comment by adejanovski** [comment]
Fixed by disabling checksum comparisons in #125
